src/components/Button/Button.stories.js

```diff
@@ -28,6 +28,7 @@ const kinds = {
   'Secondary button (secondary)': 'secondary',
   'Danger button (danger)': 'danger',
   'Ghost button (ghost)': 'ghost',
+  'Ghost Danger button (ghost-danger)': 'ghost-danger',
```

Comment:
- [ ] I recommend using standard case type here for consistency:

```suggestion
'Ghost danger button (ghost-danger)': 'ghost-danger',
```